docs/sphinx/source/api.rst

```diff
@@ -539,3 +540,32 @@ Methods for calculating back surface irradiance
    :toctree: generated/
 
    bifacial.pvfactors_timeseries
+
+
+Infinte Sheds
+=============
+
+.. autosummary::
+   :toctree: generated/
+
+   infinite_sheds.get_solar_projection_tangent
```

Comment:
pvlib python uses `get_*` for wrapper functions. `get_` should be avoided for standard functions.

Reply:
Okay just to clarify: I should remove `get_` for the base functions, but add it to method wrappers in the class `InfiniteSheds`?

Reply:
I don't really know what that class is supposed to look like, so it's hard for me to say.